templates/cust_view_appointment.html

```diff
@@ -0,0 +1,144 @@
+        <td class="td">Cancel</td>
+    </tr>
+    <tr class="tr">
+        <td class="td">Name</td>
+        <td class="td">Phone Number</td>
+        <td class="td">Email</td>
+        <td class="td">Consultation Medium</td>
+        <td class="td">Patient Type</td>
+        <td class="td">Specialist</td>
+        <td class="td">Practitioner</td>
+        <td class="td">Date</td>
+        <td class="td">Time</td>
+        <td class="td">Cancel</td>
+    </tr>
+    <tr class="tr">
+        <td class="td">Name</td>
+        <td class="td">Phone Number</td>
+        <td class="td">Email</td>
+        <td class="td">Consultation Medium</td>
+        <td class="td">Patient Type</td>
+        <td class="td">Specialist</td>
+        <td class="td">Practitioner</td>
+        <td class="td">Date</td>
+        <td class="td">Time</td>
+        <td class="td">Cancel</td>
+    </tr>
+    <tr class="tr">
+        <td class="td">Name</td>
+        <td class="td">Phone Number</td>
+        <td class="td">Email</td>
+        <td class="td">Consultation Medium</td>
+        <td class="td">Patient Type</td>
+        <td class="td">Specialist</td>
+        <td class="td">Practitioner</td>
+        <td class="td">Date</td>
+        <td class="td">Time</td>
+        <td class="td">Cancel</td>
+    </tr>
+    <tr class="tr">
+        <td class="td">Name</td>
+        <td class="td">Phone Number</td>
+        <td class="td">Email</td>
+        <td class="td">Consultation Medium</td>
+        <td class="td">Patient Type</td>
+        <td class="td">Specialist</td>
+        <td class="td">Practitioner</td>
+        <td class="td">Date</td>
+        <td class="td">Time</td>
+        <td class="td">Cancel</td>
+    </tr>
+    <tr class="tr">
+        <td class="td">Name</td>
+        <td class="td">Phone Number</td>
+        <td class="td">Email</td>
+        <td class="td">Consultation Medium</td>
+        <td class="td">Patient Type</td>
+        <td class="td">Specialist</td>
+        <td class="td">Practitioner</td>
+        <td class="td">Date</td>
+        <td class="td">Time</td>
+        <td class="td">Cancel</td>
+    </tr>
+    <tr class="tr">
+        <td class="td">Name</td>
+        <td class="td">Phone Number</td>
+        <td class="td">Email</td>
+        <td class="td">Consultation Medium</td>
+        <td class="td">Patient Type</td>
+        <td class="td">Specialist</td>
+        <td class="td">Practitioner</td>
+        <td class="td">Date</td>
+        <td class="td">Time</td>
+        <td class="td">Cancel</td>
+    </tr>
+    <tr class="tr">
+        <td class="td">Name</td>
+        <td class="td">Phone Number</td>
+        <td class="td">Email</td>
+        <td class="td">Consultation Medium</td>
+        <td class="td">Patient Type</td>
+        <td class="td">Specialist</td>
+        <td class="td">Practitioner</td>
+        <td class="td">Date</td>
+        <td class="td">Time</td>
+        <td class="td">Cancel</td>
+    </tr>
```

Comment:
Try to code in DRY format. we can use jina here right? 
eg: 
`
{% for i in range(10) %}
....
{% endfor %}
`
i see only headings no data is coming from backend.!